vss-extension.json

```diff
@@ -110,50 +110,75 @@
             "targets": [
                 "ms.vss-endpoint.endpoint-types"
             ],
-            "properties": {
-                "name": "VstsMarketplacePublishing",
-                "displayName": "Marketplace",
-                "url": {
-                    "displayName": "Marketplace URL",
-                    "value": "https://marketplace.visualstudio.com"
-                },
-                "authenticationSchemes": [
-                    {
-                        "type": "ms.vss-endpoint.endpoint-auth-scheme-basic",
-                        "inputDescriptors": [
-                            {
-                                "id": "username",
-                                "name": "Username",
-                                "description": "Username",
-                                "inputMode": "textbox",
-                                "isConfidential": false,
-                                "validation": {
-                                    "isRequired": false,
-                                    "dataType": "string",
-                                    "maxLength": 300
-                                },
-                                "values": {
-                                    "inputId": "username",
-                                    "isDisabled": true,
-                                    "defaultValue": ""
-                                }
-                            },
-                            {
-                                "id": "password",
-                                "name": "Personal access token",
-                                "description": "Team Services personal access token. Requires at least the Marketplace (publish) scope.",
-                                "inputMode": "passwordbox",
-                                "isConfidential": true,
-                                "validation": {
-                                    "isRequired": true,
-                                    "dataType": "string",
-                                    "maxLength": 300
-                                }
-                            }
-                        ]
+          "properties": {
+            "name": "VstsMarketplacePublishing",
+            "displayName": "Marketplace",
+            "url": {
+              "displayName": "Marketplace URL",
+              "value": "https://marketplace.visualstudio.com"
+            },
+            "helpMarkDown": "To publish to the Visual Studio marketplace, choose <b>Token Based Authentication</b>.<br />To publish to a Team Foundation Server 2015 Gallery, choose <b>Basic Authentication</b>",
```

Comment:
Probably, but this just works:
![image](https://cloud.githubusercontent.com/assets/4173387/16090363/6504a646-332f-11e6-9cbd-549d9449861c.png)

I'm putting the "values" items back in, otherwise the Basic Auth code pre-fills everything with admin. can you file a bug on that?